test/templates/test_discovery-territorial_lord.wgt

```diff
@@ -0,0 +1,59 @@
+[global]
+map="maps/Lesser_Ring.wmf"
+multiplayer="true"
+interactive_player="0"
+
+win_condition="territorial_lord.lua"
+
+# Territorial lord is not time based, but we can use this in the script for the timeout.
+win_condition_duration="3"
+
+# Skip 20 minute delay
+peaceful="true"
+
+fogless="false"
+custom_starting_positions="false"
```

Comment:
above are default values. Should we include them in each file?

Reply:
I think it's best to have a controlled test environment for the "real" tests. But I added a test that leaves everything to the defaults. Will that do?